doc/activities/WP5.3-Development-of-connectors-by-WP-partners-starting-from-specifiactions-for-reference-connector/WP5.3.1 - Management of development.md

```diff
@@ -3,28 +3,27 @@
-| Learning Agreement API                      |   -   |   -   |   -   |   -   |   -   |   -   |   -   |   -    |   -   |
-| **June 2017**                               |       |       |       |       |       |       |       |        |       | 
-| Transcript of Record                        |   -   |   -   |   -   |   -   |   -   |   -   |   -   |   -    |   -   |
-| **May - June 2017**                         |       |       |       |       |       |       |       |        |       | 
-| Courses                                     |:star: |   -   |   -   |   -   |   -   |   -   |:soon: |   -    |   -   |
-| **July - September 2017**                   |       |       |       |       |       |       |       |        |       | 
-| Testing and improving                       |   -   |   -   |   -   |   -   |   -   |   -   |   -   |   -    |   -   |
-| IIA                                         |:star: |   -   |   -   |   -   |   -   |   -   |:star: |:star:  |   -   |
-| Mobilities                                  |:star: |   -   |   -   |   -   |   -   |   -   |:soon: |:star:  |   -   |
-| Simple course Replication (optional)        |   -   |   -   |   -   |   -   |   -   |   -   |   -   |   -    |   -   |
+| API/Partner                                                        | Umeå   | Gent   | Porto  | SOP    | KION   | Sigma  | Oslo   | Warsaw | Others |
+|--------------------------------------------------------------------|:------:|:------:|:------:|:------:|:------:|:------:|:------:|:------:|:------:|
+| **DECEMBER 2016**                                                  |        |        |        |        |        |        |        |        |        |
+| In Registry                                                        | :link: | :link: | :soon: | :link: | :soon: | :link: | :link: | :link: | -      |
+| [Discovery API][discovery-api]                                     | :star: | :star: | :soon: | :star: | :soon: | :star: | :star: | :star: | -      |
+| [Echo API][echo-api]                                               | :star: | :star: | :soon: | :star: | :soon: | :star: | :star: | :star: | -      |
+| **MARCH 2017**                                                     |        |        |        |        |        |        |        |        |        |
+| [Institutions API][institutions-api]                               | :star: | :star: | -      | :star: | -      | :soon: | :star: | :star: | -      |
+| [Organizational Units API][ounits-api]                             | :star: | :star: | -      | -      | -      | -      | :star: | :star: | -      |
+| **APRIL 2017**                                                     |        |        |        |        |        |        |        |        |        |
+| Nominations part of [Outgoing Mobilities API][mobilities-api]      | -      | -      | -      | -      | -      | -      | -      | :soon: | -      |
```

Comment:
I didn't understand "Nominations API" . That is why I wrote :soon: but gave a :star: in "Mobilities". It should be :star: for Warsaw in this row. :)